src/s3/args.rs

```diff
@@ -1026,22 +1026,22 @@ impl<'a> ListObjectsV1Args<'a> {
 }
 
 /// Argument for [list_objects_v2()](crate::s3::client::Client::list_objects_v2) S3 API
-pub struct ListObjectsV2Args<'a> {
-    pub extra_headers: Option<&'a Multimap>,
-    pub extra_query_params: Option<&'a Multimap>,
-    pub region: Option<&'a str>,
-    pub bucket: &'a str,
-    pub delimiter: Option<&'a str>,
-    pub encoding_type: Option<&'a str>,
+pub struct ListObjectsV2Args {
```

Comment:
Can you modify `list_objects()` API to support streaming generically instead of only to ListObjectsV2 API?

Reply:
Done, PTAL @balamurugana